java/example_code/lambda/src/main/java/com/example/lambda/ListFunctions.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2010-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * This file is licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License. A copy of
+ * the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ */
+
+// snippet-comment:[These are tags for the AWS doc team's sample catalog. Do not remove.]
+// snippet-sourcedescription:[ListLambdaFunctions.java demonstrates how to list all functions specific to the account by using the AWSLambda object]
+// snippet-service:[Lambda]
+// snippet-keyword:[Java]
+// snippet-keyword:[Amazon Lambda]
+// snippet-keyword:[Code Sample]
+// snippet-sourcetype:[full-example]
+// snippet-sourcedate:[2019-11-19]
+// snippet-sourceauthor:[AWS-scmacdon]
+
+// snippet-start:[lambda.java1.ListLambdaFunctions.complete]
+package com.example.lambda;
+
+// snippet-start:[lambda.java1.list.import]
+import com.amazonaws.auth.profile.ProfileCredentialsProvider;
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.lambda.AWSLambda;
+import com.amazonaws.services.lambda.AWSLambdaClientBuilder;
+import com.amazonaws.services.lambda.model.FunctionConfiguration;
+import com.amazonaws.services.lambda.model.ListFunctionsResult;
+import com.amazonaws.services.lambda.model.ServiceException;
+import java.util.Iterator;
+import java.util.List;
+// snippet-end:[lambda.java1.list.import]
+
+
+public class ListFunctions {
+
+    public static void main(String[] args) {
+
+        // snippet-start:[lambda.java1.list.main]
+        ListFunctionsResult functionResult = null;
+
+        try {
+            AWSLambda awsLambda = AWSLambdaClientBuilder.standard()
+                    .withCredentials(new ProfileCredentialsProvider())
+                    .withRegion(Regions.US_WEST_2).build();
+
+            functionResult = awsLambda.listFunctions();
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This code might not produce accurate results if the operation returns paginated results instead of all results. Consider adding another call to check for additional results.